src/microsAuthentication/content.md

```diff
@@ -0,0 +1,1118 @@
+If your infrastructure consists of several applications interacting with each other, you might have faced the 
+issue of securing the communication between services to prevent unauthenticated requests. Imagine having three apps:
+
+1. An API
+1. A data store
+1. A secret store
+
+You might want the secret store only to reply to requests to the API and reject any request from the data store.
+
+How would the secret store decide to authenticate or deny the request?
+
+A popular approach is to request and pass identity tokens to every call within services. So instead of 
+issuing a request from the API service or the data store to the secret store directly, you might need 
+to go through an Authorisation service first, authenticate and retrieve a token that is passed along 
+to your request. That way, when the data store gets a token and presents it to the secret store, 
+it is rejected, but if the API service presents one, it will be accepted.
+
+![Three apps](image1.png)
+
+You have several options when it comes to Authorisation server:
+
+- You could use static tokens that don’t expire (and perhaps hardcoding them in the applications that 
```

Comment:
(and perhaps hardcoding them in the applications or loading them at startup from an environment variable).

The fact that the auth server is not there is implicit, I guess?
I'm trying to break down the long sentence in the parenthesis.